### PR TITLE
Few parsing and path fixes

### PR DESCRIPTION
**Pull request by RaistGH**
I've got this fixes while running the code through about 5000 vcards

### CODE REVIEW

File: vcardlib.py [hunk 1]
```diff
@@ -379,7 +379,9 @@ def select_most_relevant_name(names):
     longuest_length = 0
     pos = 0
     for name in names:
-        if not name:
+        if name == "":
+            continue
+        else:
```
[the remark this answers is not in the document]
Comment (mbideau) [reply vcardlib.py:384]: Also it makes all the tests fails... so better fix it in the PR please :wink: 
Thanks in advance

**Comment by mbideau** [comment]
Hey @RaistGH ,
Thank you very much for your fixes, very apprecitated :pray: 
As soon as you fix the two feedback/review notes I have written above I would happily merge this PR :smiley:

**Review by mbideau — CHANGES_REQUESTED** [review]

File: vcardlib.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mbideau) [reply vcardlib.py]: I've fixed it :wink:

[the remark this answers is not in the document]
Comment (RaistGH) [reply vcardlib.py]: Thank for the update! Sorry, I was away and couldn't check this to fix the PR myself.

**Pull request merged**